conda/recipes/kvikio/meta.yaml

```diff
@@ -71,7 +71,7 @@ requirements:
     - cupy >=12.0.0
     - zarr
     # See https://github.com/zarr-developers/numcodecs/pull/475
```

Comment:
(Same as dependencies.yaml) Is the PR linked above still relevant? I think it may be safe to remove this comment now.

Reply:
I presume it is still relevant as it explains why we have the minimum value clamp as well. If kvikio is combined with another package that requests 12.X we need to report as unsolvable.